### PR TITLE
[Nemo Unit Tests] Split CPU unit tests

### PR DESCRIPTION
**Pull request by pablo-garay**
Split CPU unit tests

### CODE REVIEW

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

File: .github/workflows/cicd-main.yml [hunk 1]
```diff
@@ -133,16 +133,119 @@ jobs:
...
-  #   needs: [cicd-test-container-setup]
-  #   uses: ./.github/workflows/_test_template.yml
-  #   with:
-  #     RUNNER: self-hosted-azure-cpu
-  #     TIMEOUT: 60
-  #     SCRIPT: |
-  #       CUDA_VISIBLE_DEVICES="" NEMO_NUMBA_MINVER=0.53 pytest -m "not pleasefixme" --cpu --with_downloads --relax_numba_compat
-  #     IS_OPTIONAL: true
+  # L0: CPU unit tests
+  OPTIONAL_L0_Unit_Tests_CPU_ASR:
```
Comment (ericharper) [inline .github/workflows/cicd-main.yml:137]: Can we make the the unit tests required?

**Review by ericharper — CHANGES_REQUESTED** [review]
Can we make the the unit tests required?

**Comment by pablo-garay** [comment]
image

CI passed

> Can we make the the unit tests required?

done

**Comment by pablo-garay** [comment]
CI passed https://github.com/NVIDIA/NeMo/actions/runs/10732427046/job/29771216029?pr=10365

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**